tempesta_fw/http_parser.h

```diff
@@ -85,19 +85,34 @@ typedef struct {
  * @hbh_parser	- list of special and raw headers names to be treated as
  *		  hop-by-hop
  * @_date	- currently parsed http date value;
+ * @st		- template showing in what state the i-th character in the
```

Comment:
The structure documentation is outdated and doesn't reflect exact members. E.g. there is no `st` member.